app/resources/api/v2/state_change_resource.rb

```diff
@@ -15,14 +15,87 @@ module V2
     # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
     # of the JSON:API standard.
     class StateChangeResource < BaseResource
-      immutable
+      ###
+      # Attributes
+      ###
 
-      attribute :previous_state
+      # @!attribute [rw] contents
+      #   @return [Array] Array of "contents" to fail, deciphered by the target.
+      attribute :contents
+
+      # @!attribute [w] customer_accepts_responsibility
+      #   @param value [Boolean] Sets whether the customer proceeded against advice and will still be charged in the
+      #     event of a failure.
+      #   @return [Void]
+      attribute :customer_accepts_responsibility
+
+      # @!attribute [r] previous_state
+      #   @return [String] The previous state of the target before this state change.
+      attribute :previous_state, readonly: true
+
+      # @!attribute [rw] reason
+      #   @return [String] The previous state of the target before this state change.
+      attribute :reason
+
+      # @!attribute [rw] target_state
+      #   @return [String] The state of the target after this state change.
+      #   @note This attribute is required.
       attribute :target_state
-      attribute :created_at
-      attribute :updated_at
```

Comment:
@KatyTaylor you added these about 5 years ago.  Other endpoints don't list the created at and updated at dates for resources.  I'm happy to leave these in if you think they're useful, but I wanted to align with the v1 endpoint as much as possible without upsetting anything.